### PR TITLE
Rimsha3

### PR DESCRIPTION
**Pull request by Rxmsha**

### CODE REVIEW

File: code/app/src/main/java/com/example/the_tarlords/ui/event/EventEditFragment.java [hunk 1]
```diff
@@ -210,9 +219,32 @@ public void onViewCreated(@NonNull View view, Bundle savedInstanceState) {
...
 
+        // this is for imageslider for event images
+        ViewPager2 viewPager = view.findViewById(R.id.viewPager);
+        int[] images = new int[]{R.drawable.plant, R.drawable.profile, R.drawable.participants};
+        ImageSliderAdapter adapter = new ImageSliderAdapter(images);
+        viewPager.setAdapter(adapter);
+
+        // this is for navigating to maps fragment by clicking on the map imageView5
+        ImageView imageView2 = view.findViewById(R.id.mapImage);
+        imageView2.setOnClickListener(new View.OnClickListener() {
```
Comment (br0ken-mirrorz) [inline code/app/src/main/java/com/example/the_tarlords/ui/event/EventEditFragment.java:234]: I feel like we don't need to access the map fragment while editing

File: code/app/src/main/res/drawable/participants.png
Comment (br0ken-mirrorz) [inline code/app/src/main/res/drawable/participants.png:1]: does this need to be added?

File: code/app/src/main/res/drawable/plant.jpg
Comment (br0ken-mirrorz) [inline code/app/src/main/res/drawable/plant.jpg:1]: does this need to be added?

Comment (Rxmsha) [reply code/app/src/main/res/drawable/plant.jpg:1]: no you can delete those two. They were there for the slider image thing but I took that out.

Comment (br0ken-mirrorz) [reply code/app/src/main/res/drawable/plant.jpg:1]: The slider is still in this pull request

**Pull request closed without merging**